dbt_project.yml

```diff
@@ -467,20 +467,14 @@ models:
         +schema: fraxswap_avalanche_c
         +materialized: view
 
-    mstable:
-      +schema: mstable
-      +materialized: view
-      ethereum:
-        +schema: mstable_ethereum
-        +materialized: view
-        
-    platypus_finance:
-      +schema: platypus_finance
+    woofi:
+      +schema: woofi
       +materialized: view
       avalanche_c:
-        +schema: platypus_finance_avalanche_c
+        +schema: woofi_avalanche_c
         +materialized: view
 
+
```

Comment:
Hi @LeviScoffie ,
you may mistakenly remove something during the merge.
Please make sure ['mstable', 'platypus_finance', 'woofi'] exists in the dbt_project.yml.

Reply:
```
    mstable:
      +schema: mstable
      +materialized: view
      ethereum:
        +schema: mstable_ethereum
        +materialized: view
        
    platypus_finance:
      +schema: platypus_finance
      +materialized: view
      avalanche_c:
        +schema: platypus_finance_avalanche_c
        +materialized: view
        
    woofi:
      +schema: woofi
      +materialized: view
      avalanche_c:
        +schema: woofi_avalanche_c
        +materialized: view
```